### DISCUSSION TITLE
False positive for drainage gutter

### DISCUSSION THREAD

**Discussion by pszafer** [blakeblackshear/frigate#3866] [Ask A Question]
Hi, I get all the time false positive for my gutter detected as person. It sometimes shows detection of the person at 86% level.
![image](https://user-images.githubusercontent.com/690510/190955702-bada3523-b029-4f1e-b7ac-828aa0e7e120.png)

So I made special zone for that gutter 
![image](https://user-images.githubusercontent.com/690510/190955887-08fd9593-d1f5-4d2c-8fa5-ffafb85c2dca.png)
and there set person detection:
```
   filters:
          person:
            min_area: 20000
```

But for some reason Frigate doesn't recognize that this "person" is in this zone, it doesn't set `current_zones` or `entered_zones` at all. Any ideas?

**Comment by NickM-27** [comment]
Why would you want to create a zone for a false positive person?

If you do want to ignore that then the recommended strategy would be using a min_area filter at the camera level.

**Reply by pszafer** [reply]
if person is closer to camera then min area is different than if person is far away from it

**Reply by NickM-27** [reply]
Yes but that false positive isn't moving, so min_area will always filter it out (just make it small enough to not filter out a real person)

**Comment by blakeblackshear** [comment]
Zone filters only effect presence in the zone. They do not filter out false positives.

**Reply by pszafer** [reply]
I wanted to add zones so I can filter it out in mqtt event

**Reply by NickM-27** [reply]
> I wanted to add zones so I can filter it out in mqtt event

Zones are for limiting events to areas of interest, not for false positives. 

Check out the docs https://docs.frigate.video/guides/false_positives
